TeamCode/src/main/java/org/firstinspires/ftc/teamcode/components/LatchSystem.java

```diff
@@ -4,45 +4,68 @@
 
 
 public class LatchSystem {
-    private Servo servoLeft;
-    private Servo servoRight;
-    public boolean latched;
+    private Servo leftServo;
+    private Servo rightServo;
+    public boolean rightLatched;
+    public boolean leftLatched;
     private final double LEFT_DOWN_POSITION = 0.714623491755917;
```

Comment:
If we maintain consistency, and define the servos in an EnumMap, the Enum keys could also be used to define these constants.

Reply:
Something ala:
```
    public enum Latch {
        LEFT (0.715, 0.446),
        RIGHT (0.189, 0.456);

        private final double upPosition;
        private final double downPosition;

        Latch(double downPosition, double upPosition) {
            this.downPosition = downPosition;
            this.upPosition = upPosition;
        }

        private double upPosition() {
            return upPosition;
        }

        private double downPosition() {
            return downPosition;
        }
    }

    public EnumMap<Latch, Servo> latches;

    private void initServo() {
        latches.forEach((name, servo) -> {
            servo.setPosition(name.upPosition());
        });
    }
```